kie-server-parent/kie-server-services/kie-server-services-kafka/src/main/java/org/kie/server/services/jbpm/kafka/KafkaServerProducer.java

```diff
@@ -75,7 +75,12 @@ private void sendEvent(ProcessInstance processInstance,
                            String name,
                            Object value) {
         if (producerReady.compareAndSet(false, true)) {
-            producer = producerSupplier.get();
+            try {
+                producer = producerSupplier.get();
+            }
+            catch (Exception ex) {
+                producerReady.set(false);
```

Comment:
Shouldn't the exception be thrown again? Or at least, return from this method.
As the producer is not ready will fail in the send call.

Reply:
yes, this was caused by a problem in my ide